src/bin.ts

```diff
@@ -3,50 +3,14 @@
 import { argv } from 'yargs';
 import path from 'path';
 import runVrt from './index';
```

Comment:
import runVrt from './index';
import { VRTCommandOptions } from './index';

can be:

import runVrt, { VRTCommandOptions } from './index';

also `madge --circular src/bin.ts`

Found 1 circular dependency:

1) index.ts > report.ts

could you please update it?